Updating DLP Policy to Common Controls (#469)
cisagov/ScubaGoggles: baselines/commoncontrols.md

- [Data Regions](#15-data-regions-and-storage)
- [Additional Google Services](#16-additional-google-services)
- [Multi-Party Approvals](#17-multi-party-approval)

> - [Data Protection](#18-dlp-rules)

Author: Data protection is what Google calls it, maybe since we allow for 3rd party DLP maybe we adjust it to be Data Loss Prevention as the title in TOC?

2.	Navigate to **Security** -> **Authentication** -> **Multi-party approval settings**.
3.	Ensure **Require multi party approval for sensitive admin actions** is checked.
4.	Click **Save**.

## 18. DLP rules

> This recommendation applies only to agencies that allow external sharing (see section 2.1).

Reviewer: Actually, I think we'd just want to take this line out entirely. I think we want them to have DLP in place regardless. If they also disable external sharing entirely, great, now they're extra protected - defense in depth.

> This recommendation applies only to agencies that allow external sharing (see section 2.1).

Author: Removed this line

  - [T1213: Data from Information Repositories](https://attack.mitre.org/techniques/T1213/)

### Resources
- [GWS Admin Help \| Protect sensitive information using DLP](https://support.google.com/a/topic/7556687?hl=en&ref_topic=7558840&fl=1&sjid=4459086914710819343-NA)
- [GWS Admin Help \| Use Workspace DLP to prevent data loss](https://support.google.com/a/answer/9646351?hl=en&visit_id=638635679011849528-69139467&ref_topic=9646660&rd=1)
- [GWS Admin Help \| Create DLP for Drive rules and custom content detectors](https://support.google.com/a/answer/9655387)
- [GWS Admin Help \| Prevent data leaks from Chat messages & attachments](https://support.google.com/a/answer/10846568)
- [GWS Admin Help \| Prevent data leaks in email & attachments](https://support.google.com/a/answer/14767988?fl=1&sjid=4620103790740920406-NA)

### Prerequisites
- None

### Implementation

#### GWS COMMONCONTROLS 18 Common Instructions
1.  Sign in to the [Google Admin Console](https://admin.google.com).
2.  Select **Menu -\> Security -\> Access and data control -\> Data protection**.

> 3.  Click **Manage Rules**. 

Author: Updated the location for clarifty

4.  For each DLP rule necessary to meet the standards of an agency:
5.  Click **Add rule** -\> **New rule** or click **Add rule** -\> **New rule from template**. For templates, select a template from the Templates page.

#### GWS.COMMONCONTROLS.18.1v0.3 Instructions
1. For new rules:
2. In the **Name** section, add the name and description of the rule.
3. In the **Scope** section, apply this rule only to the entire domain or to selected organizational units or groups, and click **Continue**. If there's a conflict between organizational units and groups in terms of inclusion or exclusion, the group takes precedence.
4. In the **Apps** section, choose the trigger for **Google Drive, Drive files** then click **Continue**.
5. In the **Conditions** section, click **Add Condition**.

> 6. Configure appropriate content and condition definition(s) based upon the agency's individual requirements and click **Continue**.

Reviewer 2: Unsure why the "context conditions" section didn't show up when we went through the settings, but I think we can exclude this for the scope of this issue.

> 7. In the **Actions** section, select the appropriate action to warn or block sharing for Google Chat, Gmail, and Google Drive based upon the agency's individual requirements.

Reviewer 2: Same question for the Actions section, is there prescriptive guidance on whether "block external sharing", "warn on external sharing", "disable download, print, and copy for commenters and viewers", or "apply drive labels" must be configured?

 
![Screenshot (253)](https://github.com/user-attachments/assets/b8b8369f-e882-4b18-888f-4a0f73ef9a69)

Author: added in actions section policy and implementation steps

4. In the **Conditions** section:
    1. Click **Action Condition**. For **Content type to scan** select **All content**. For **What to scan for** select **Matches predefined data type**. For **Select data type** select **Global - Credit card number**. Select the remaining condition properties according to agency need.
    2. Click **Action Condition**. For **Content type to scan** select **All content**. For **What to scan for** select **Matches predefined data type**. For **Select data type** select **United States - Individual Taxpayer Indentification Number**. Select the remaining condition properties according to agency need. 
    3. Click **Action Condition**. For **Content type to scan** select **All content**. For **What to scan for** select **Matches predefined data type**. For **Select data type** select **United States - Social Security Number***. Select the remaining condition properties according to agency need. 
    4. Configure other appropriate content and condition definition(s) based upon the agency's individual requirements and click **Continue**.

> 5. In the **Actions** section, select the appropriate action to warn or block sharing for Google Chat based upon the agency's individual requirements (See [GWS.COMMONCONTROLS.18.4v0.3](#gwscommoncontrols184v03) ).

Reviewer: Recommend change for consistency with the 18.4 policy requirement.
```suggestion
5. In the **Actions** section, select **Block**. Under **Select when this action should apply**, select **External Conversations**, **Spaces**, **Group chats**, and **1:1 chats** (See [GWS.COMMONCONTROLS.18.4v0.3](#gwscommoncontrols184v03)).
```

3. In the **Apps** section, choose the trigger for **Gmail, Message sent** then click **Continue**.
4. In the **Conditions** section:
    1. Click **Action Condition**. For **Content type to scan** select **All content**. For **What to scan for** select **Matches predefined data type**. For **Select data type** select **Global - Credit card number**. Select the remaining condition properties according to agency need.
    2. Click **Action Condition**. For **Content type to scan** select **All content**. For **What to scan for** select **Matches predefined data type**. For **Select data type** select **United States - Individual Taxpayer Indentification Number**. Select the remaining condition properties according to agency need. 
    3. Click **Action Condition**. For **Content type to scan** select **All content**. For **What to scan for** select **Matches predefined data type**. For **Select data type** select **United States - Social Security Number***. Select the remaining condition properties according to agency need. 
    4. Configure other appropriate content and condition definition(s) based upon the agency's individual requirements and click **Continue**.

> 5. In the **Actions** section, select the appropriate action to warn or block sharing for Gmail based upon the agency's individual requirements (See [GWS.COMMONCONTROLS.18.4v0.3](#gwscommoncontrols184v03) ).

Reviewer: Recommended change for consistency with the 18.4 policy requirement.
```suggestion
5. In the **Actions** section, select **Block message**. Under **Select when this action should apply**, check **Messages sent to external recipients** (See [GWS.COMMONCONTROLS.18.4v0.3](#gwscommoncontrols184v03)).
```

2.	Navigate to **Security** -> **Authentication** -> **Multi-party approval settings**.
3.	Ensure **Require multi party approval for sensitive admin actions** is checked.
4.	Click **Save**.

## 18. DLP Rules

Using data loss prevention (DLP), organizations can create and apply rules to control the content that users can share in files outside the organization. DLP gives you control over what users can share and prevents unintended exposure of sensitive information.

DLP rules can use predefined content detectors to match PII (e.g., SSN), credentials (e.g., API keys), or specific document types (e.g., source code). Custom rules can also be applied based upon regex match or document labels.

### Policies
#### GWS.COMMONCONTROLS.18.1v0.3

A custom policy SHALL be configured for Google Drive to protect PII and sensitive information, as defined by the agency. At a minimum, credit card numbers, U.S. Individual Taxpayer Identification Numbers (ITIN), and U.S. Social Security numbers (SSN) SHALL be blocked.

> - _Rationale:_ Data Loss Prevention (DLP) rules help identify and limit the sharing of sensitive content, protecting agency information. Blocking and/or having warnings on these DLP-scanned files from being shared with users can reduce the risk of unintentional introduction of sensitive content.

Author: I think mirroring scuba gear makes the most sense.

- [GWS Admin Help \| Prevent data leaks from Chat messages & attachments](https://support.google.com/a/answer/10846568)
- [GWS Admin Help \| Prevent data leaks in email & attachments](https://support.google.com/a/answer/14767988?fl=1&sjid=4620103790740920406-NA)

### Prerequisites
- None

### Implementation

#### GWS COMMONCONTROLS 18 Common Instructions
1.  Sign in to the [Google Admin Console](https://admin.google.com).
2.  Select **Menu -\> Security -\> Access and data control -\> Data protection**.
3.  Under Data protection rules and detectors click **Manage Rules**. 
4.  For each DLP rule necessary to meet the standards of an agency:
5.  Click **Add rule** -\> **New rule** or click **Add rule** -\> **New rule from template**. For templates, select a template from the Templates page.

#### GWS.COMMONCONTROLS.18.1v0.3 Instructions
1. For new rules:
2. In the **Name** section, add the name and description of the rule.
3. In the **Scope** section, apply this rule to the entire domain and click **Continue**.
4. In the **Apps** section, choose the trigger for **Google Drive, Drive files** then click **Continue**.
5. In the **Conditions** section, click **Add Condition**.

>     1. Select **All content** then **Matches predefined data type** then **Global - Credit card number** then **very low** and enter 1 for **Minimum unique matches** **minimum match count**.

Reviewer: A few things,
- If you remove "Add Conditions" from above, be sure to add it here
- Maybe spell out the field names instead of assuming the order?
- Since the baseline does not require or recommend a specific threshold, maybe leave that open here.

What do think about this:
```suggestion
    1. Click **Action Condition**. For **Content type to scan** select **All content**. For **What to scan for** select **Matches predefined data type**. For **Select data type** select **Global - Credit card number**. Select the remaining condition properties according to agency need.
```

I won't spell it out, but if you like those suggestions, be sure to add that for the other conditions.

Reviewer: @mdueltgen My bad, my suggestion had a typo. Instead of "Action Condition" it should say "Add Condition." Since it's my typo, I'm can fix it if you'd like.

    2. Select **All content** then **Matches predefined data type** then **United States - Individual Taxpayer Indentification Number** then **very low** and enter 1 for **Minimum unique matches** **minimum match count**.
    3. Select **All content** then **Matches predefined data type** then **United States - Social Security Number** then **very low** and enter 1 for **Minimum unique matches** **minimum match count**.
    4. Configure other appropriate content and condition definition(s) based upon the agency's individual requirements and click **Continue**.

> 6. In the **Actions** section, select the appropriate action to block external sharing for Google Drive based upon the agency's individual requirements (See [GWS.COMMONCONTROLS.18.4v0.3](#gwscommoncontrols184v03) ).

Reviewer: With the inclusion of 18.4, isn't there only one "appropriate action" here?
```suggestion
6. In the **Actions** section, select **Block external sharing** (per [GWS.COMMONCONTROLS.18.4v0.3](#gwscommoncontrols184v03)).
```
